xbmc/addons/AddonStatusHandler.cpp

```diff
@@ -38,8 +38,13 @@ namespace ADDON
 
 CCriticalSection CAddonStatusHandler::m_critSection;
 
-CAddonStatusHandler::CAddonStatusHandler(const std::string &addonID, ADDON_STATUS status, std::string message, bool sameThread)
-  : CThread(("AddonStatus " + addonID).c_str()),
+CAddonStatusHandler::CAddonStatusHandler(const std::string& addonID,
+                                         uint32_t instanceID,
+                                         ADDON_STATUS status,
+                                         std::string message,
+                                         bool sameThread)
+  : CThread(("AddonStatus " + addonID + "@" + std::to_string(instanceID)).c_str()),
```

Comment:
would it make more sense to switch addonid and instanceid?

instance@addon ("instance at addon") reads more natural, a bit like user@server in URLs we all know.